### ISSUE TITLE
Remove game logic from index.js

### ISSUE THREAD

**Issue by linkret** [linkret/CrosseyCircley#12]
Currently the entirety of the tic-tac-toe game logic is implemented inside index.js, which is lazy and unelegant. It would be a good idea to seperate it into another one or two javascript modules, so index.js could be cleaned up a lot.

**Comment by hannesfant** [comment]
Sweet!

You should probably add the hacktoberfest label to your repo to try and get some more people to help out with the issues :)

**Comment by linkret** [comment]
Thanks, I've never even heard of hacktoberfest. :)

**Comment by hannesfant** [comment]
Yeah it's pretty wild haha. As soon as you tag a repo with it, especially issues, you just get absolutely flooded with people that want to help. Did it last year on one of my projects and people starting popping up left and right

**Comment by linkret** [comment]
Cool, I added the topic to the repo, and a tag to some of the issues.
